scripts/helper-update-version

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
+ # Retrieve sources, get sha256sum, parse package version and update these strings in manifest, README and /conf/app.src
+ 
+url=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
+wget -nc --quiet $url -P ./tmp
+CHECKSUM=$(sudo sha256sum ./tmp/cesium-*-web.zip | head -c 64)
+sed -i "s/SOURCE_SUM=.*/SOURCE_SUM=${CHECKSUM}/" ../conf/app.src
+
+PACKAGE_VERSION=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "tag_name" | head -1 | cut -d\" -f4)
+sed -i "s#SOURCE_URL=.*#SOURCE_URL=https://github.com/duniter/cesium/releases/download/${PACKAGE_VERSION}/cesium-${PACKAGE_VERSION}-web.zip#" ../conf/app.src
+sed -i "s#\*\*Shipped version:\*\*.*#\*\*Shipped version:\*\* ${PACKAGE_VERSION}#" ../README.md
+sed -i "s#    \"version\": \".*#    \"version\": \"${PACKAGE_VERSION}\~ynh1\",#" ../manifest.json
+
+sudo rm -f -R ./tmp
+
+git push
+git request-pull
```

Comment:
You can automatically commit that way:
```bash
git commit README.md manifest.json conf/app.src  -m "v$VERSION"
```

I gave you the rights on this repository.
What are you trying to achieve this this two commands?

Reply:
I guess I wanted to do that but as I didn't have the rights, I wanted to do it on my repo and send you a PR at the same time.
Thank you for the rights!
`git commit README.md manifest.json conf/app.src  -m "$PACKAGE_VERSION"`
I changed `"v$VERSION"` to `"$PACKAGE_VERSION"`

Reply:
Done